pulpcore/app/util.py

```diff
@@ -159,3 +162,25 @@ def get_request_without_query_params(context):
         request.query_params._mutable = False
 
     return request
+
+
+def verify_signature(filepath, public_key, pubkey_fingerprint=None):
+    """Check whether the provided file can be verified with the particular public key."""
+    with tempfile.TemporaryDirectory() as temp_directory_name:
```

Comment:
```suggestion
    with tempfile.TemporaryDirectory(dir=".") as temp_directory_name:
```
This can be run inside tasks and they need to use the current working directory for temporary stuff.